### PR TITLE
Tb fixes and improvements

### PR DESCRIPTION
**Pull request by jbrezmorf**

### CODE REVIEW

File: src/genie/genie_ert.py [hunk 1]
```diff
@@ -13,6 +13,10 @@
```
Comment (jbrezmorf) [inline src/genie/genie_ert.py:14]: Need to apply to Genie ST as well.

File: src/genie/ui/dialogs/color_maps/black_blue_white.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+[
```
Comment (jbrezmorf) [inline src/genie/ui/dialogs/color_maps/black_blue_white.json:1]: Keep only subset of paraview color maps:
viridis
cold hot
cold to warm
jet
linear green
rainbow desauturated

**Review by jbrezmorf — COMMENTED** [review]
Use in Genie ST, remove some colormaps.
Review from Radek.

File: src/genie/ui/dialogs/color_map_editor.py [hunk 3]
```diff
@@ -0,0 +1,196 @@
+import glob
```
Comment (jbrezmorf) [inline src/genie/ui/dialogs/color_map_editor.py:1]: Remove unfinished code. Tag the branch, make an issue for possible future usage.

Comment (TomasBlazek1) [reply src/genie/ui/dialogs/color_map_editor.py:1]: This is where the color map is selected/imported. Should it be renamed?

**Comment by radeksrb** [comment]
After merge master not working.

/home/radek/envs/genie/bin/python genie/genie_ert.py
Traceback (most recent call last):
  File "/home/radek/work/Genie/src/genie/genie_ert.py", line 7, in <module>
    from genie.ui.main_window import MainWindow
  File "/home/radek/work/Genie/src/genie/ui/main_window.py", line 2, in <module>
    from .tab_widget import TabWidget
  File "/home/radek/work/Genie/src/genie/ui/tab_widget.py", line 3, in <module>
    from .tabs.view_3d import View3D
  File "/home/radek/work/Genie/src/genie/ui/tabs/view_3d.py", line 6, in <module>
    from ..view_3d.utility import current_inv_colormap_filename
ModuleNotFoundError: No module named 'genie.ui.view_3d.utility'

**Pull request merged**